perun-openapi/openapi.yml

```diff
@@ -4737,6 +4737,296 @@ paths:
+                attrNames:
+                  type: array
+                  items:
+                    type: string
+                  default: []
+
+  /json/groupsManager/createGroup/v-n-d:
+      post:
+        tags:
+          - GroupsManager
+        operationId: createGroupWithVoNameDescription
+        description: Creates a new group in the specific VO.
+        responses:
+          '200':
+            $ref: '#/components/responses/GroupResponse'
+          default:
+            $ref: '#/components/responses/ExceptionResponse'
+        requestBody:
+          required: true
+          content:
+            application/json:
+              schema:
+                title: CreateGroup
+                description: "input to createGroup"
+                type: object
+                required:
+                  - vo
+                  - name
+                  - description
+                properties:
+                  vo: {type: integer, description: "vo id", nullable: false}
+                  name: {type: string}
+                  description: {type: string}
+
+  /json/groupsManager/createGroup/pg-n-d:
+    post:
+      tags:
+        - GroupsManager
+      operationId: createGroupWithParentGroupNameDescription
+      description: Creates a subgroup of a group.
+      responses:
+        '200':
+          $ref: '#/components/responses/GroupResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: CreateGroup
+              description: "input to createGroup"
+              type: object
+              required:
+                - parentGroup
+                - name
+                - description
+              properties:
+                parentGroup: {type: integer, description: "parentGroup id", nullable: false}
+                name: {type: string}
+                description: {type: string}
+
+  /json/groupsManager/deleteGroups:
+    post:
+      tags:
+        - GroupsManager
+      operationId: deleteGroups
+      summary: Forcefully deletes a list of groups (remove all group members, remove group from resources).
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: DeleteGroups
+              description: "input to deleteGroups"
+              type: object
+              required:
+                - groups
+                - forceDelete
+              properties:
+                groups:
+                  type: array
+                  items:
+                    type: integer
+                forceDelete: {type: boolean}
+
+  /json/groupsManager/moveGroup/dg-mg:
```

Comment:
Again, URL parameters instead of request body would make the generated Java API much simpler.